docs/guides/cookie-monitoring.mdx

```diff
@@ -44,7 +44,46 @@ These are end-to-end examples in various programming languages.
 
 <Accordion title="Python SDK Example">
 ```python
-# TODO
+import os
+import time
+
+from flareio import FlareApiClient
+
+
+api_key: str | None = os.environ.get("FLARE_API_KEY")
```

Comment:
Can you remove this in favour of:
```
api_client = FlareApiClient(
    api_key=os.environ["FLARE_API_KEY"],
)
```

Which provides a similar error and is simpler ?